lib/src/sidenav/Sidenav.tsx

```diff
@@ -19,12 +20,12 @@ const DxcSidenav = ({ padding, children }) => {
   );
 };
 
-const Title = ({ children }) => <SideNavMenuTitle>{children}</SideNavMenuTitle>;
+const Title = ({ children }: SidenavTitlePropsType): JSX.Element => <SideNavMenuTitle>{children}</SideNavMenuTitle>;
 
-const Subtitle = ({ children }) => <SideNavMenuSubTitle>{children}</SideNavMenuSubTitle>;
+const Subtitle = ({ children }: SidenavSubtitlePropsType): JSX.Element => <SideNavMenuSubTitle>{children}</SideNavMenuSubTitle>;
 
-const Link = ({ href, onClick, children }) => (
-  <SideNavMenuLink href={href} onClick={onClick}>
+const Link = ({ tabIndex, href, onClick, children }: SidenavLinkPropsType): JSX.Element => (
```

Comment:
At the component props, give `tabIndex` an initial value of 0 to remove the bitwise OR operator below.